SPECS/kernel-hci/kernel-hci.spec

```diff
@@ -136,11 +136,12 @@ Requires:       audit
 %description tools
 This package contains the 'perf' performance analysis tools for Linux kernel.
 
-%package -n     python3-perf
+%package        python3-perf
```

Comment:
I think naming this as python3-perf-hci would be more consistent with out kernel variant naming scheme than kernel-hci-python3-perf. That way, kernel would be to python3-perf as kernel-hci would be to python3-perf-hci.

Further, there's a good reason why kernel's python3-perf and bpftool subpackages aren't normally prefixed with "kernel-". In the case of python3-perf, it's because it's fundamentally a python3 module. Likewise, we shouldn't do this for our kernel variants.